### PR TITLE
Fix typos found by codespell.

### PR DESCRIPTION
**Pull request by matthiaskrgr**

### CODE REVIEW

File: src/ci/docker/scripts/musl.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ shift
 
 export CFLAGS="-fPIC $CFLAGS"
 
-# FIXME: remove the patch when upate to 1.1.20
+# FIXME: remove the patch when update to 1.1.20
```
Comment (varkor) [inline src/ci/docker/scripts/musl.sh]: "when updating to 1.1.20" sounds better.

File: src/librustc/hir/map/definitions.rs [hunk 2]
```diff
@@ -9,7 +9,7 @@
 // except according to those terms.
 
 //! For each definition, we track the following data.  A definition
-//! here is defined somewhat circularly as "something with a def-id",
+//! here is defined somewhat circular as "something with a def-id",
```
Comment (varkor) [inline src/librustc/hir/map/definitions.rs]: It was correct before.

File: src/librustc_mir/dataflow/mod.rs [hunk 3]
```diff
@@ -605,7 +605,7 @@ pub trait BitDenotation: BitwiseOperator {
     /// `sets.on_entry` to that local clone into `statement_effect` and
     /// `terminator_effect`).
     ///
-    /// When its false, no local clone is constucted; instead a
+    /// When its false, no local clone is constructed; instead a
```
Comment (varkor) [inline src/librustc_mir/dataflow/mod.rs]: Should be "it's".

**Review by varkor — COMMENTED** [review]
Could you squash the commits? Once the couple of comments have been addressed, I'll approve, thanks!

File: src/librustc_mir/dataflow/mod.rs (hunk 3 above)
Comment (matthiaskrgr) [reply src/librustc_mir/dataflow/mod.rs]: all done, thanks

**Comment by varkor** [comment]
Thanks!

@bors r+ rollup

**Pull request merged**